src/config.cc

```diff
@@ -357,10 +358,11 @@ void Config::initFieldCallback() {
         if (args.size() != 2) return Status(Status::NotOK, "wrong number of arguments");
         if (args[0] != "no" && args[1] != "one") {
           master_host = args[0];
-          master_port = std::atoi(args[1].c_str());
-          if (master_port <= 0 || master_port >= 65535) {
+          auto parse_result = ParseInt<int>(args[1].c_str(), NumericRange<int>{1, 65535 - 1}, 10);
```

Comment:
Extract `65535` as a constant by the way?

Reply:
@caipengbo  Hi, May I ask where do you think define the constants are appropriate? Do you think it is appropriate in `config.h`?

Reply:
I think in config.h is a good idea